sklearn/sklearn-template/template/trainer/task.py

```diff
@@ -51,10 +51,10 @@ def _train_and_evaluate(estimator, dataset, output_dir):
 
   # Write model and eval metrics to `output_dir`
   model_output_path = os.path.join(
-      output_dir, 'models', metadata.MODEL_FILE_NAME)
```

Comment:
Have you tried to deployed it ? I remember the deployment of the scikit-learn model require that there should be only on single file in the pointed folder with name model.joblib.